src/agents/common.jl

```diff
@@ -171,6 +193,7 @@ function HEMData(input_filename::String; epsilon::AbstractFloat = 1.0E-3)
         index_z,
         index_sector,
         h_to_sector, 
+        county_to_ba,
```

Comment:
Again, "county" is too project-specific. Perhaps what's needed is for an "index_h_z_mapping.csv" to be made in HEMData.jl similar to "index_h_sector_mapping.csv"?